groups/iam/module-iam-policies/test-data-generator-stack-terraform-apply.tf

```diff
@@ -0,0 +1,70 @@
+data "aws_iam_policy_document" "test_data_generator_stack_terraform_apply" {
```

Comment:
I would rename these to just `apply`, `destroy` and `plan`

Reply:
@Jata2k - did you mean to rename the file or the data block (or both)?

Reply:
I mean I would change it to `data "aws_iam_policy_document" "apply"`. You already have the context from the repo and module as a whole. There's no need to reiterate it

Reply:
Gotcha. Will do. 👍

Reply:
This still needs addressing